client/app/scripts/liveblog-edit/directives.js

```diff
@@ -534,15 +542,74 @@ define([
                 }
             };
         }])
-        .directive('freetypeLink', ['$compile', function($compile) {
+        .directive('freetypeText', ['$templateCache', function($templateCache) {
 
             return {
                 restrict: 'E',
-                template: '<input type="url" ng-model="link"/>',
-                controller: function() {
-                },
+                template: $templateCache.get('scripts/liveblog-edit/views/freetype-text.html'),
```

Comment:
I still don't understand what that template cache is used for and why not using the standard approach to load a template.
Plus this will pose problem in the webpack branch: https://github.com/liveblog/liveblog/commit/492fc5ad993f3e368bcefeb43cae8123f5cff5e3